content/contributor/content/overview.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "Overview"
+sidebartitle: Overview
+url: "/contributor/content/overview"
+weight: 1
+---
+
+Everything you can find inside the virtual world of Decentraland is hosted on a distributed network of servers known as Catalysts, that provide (among other things) the content API. They are the means by which clients upload and download content.
+
+The complete API of the Catalyst is available in [its own reference](https://decentraland.github.io/catalyst-api-specs/). Here, we'll focus on understanding how content is organized, and how it can be discovered and downloaded.
+
+All of the important concepts mentioned below are further described in their own pages.
+
+## Files, Entities, Pointers
+
+There are three layers in Decentraland's content system: [files]({{< relref "filesystem" >}}) are packaged inside [entities]({{< relref "entities" >}}), and entities are discovered using [pointers]({{< relref "pointers" >}}).
+
+_!!Placeholder GoAT diagram, this is a necessary complement to the text and needs to be better_
```

Comment:
We can implement mermaid

Reply:
I tried Mermaid over the weekend, and maybe we'd be better off with a static image. 

My first and foremost wanted feature was the ability to add subtitles to each box, in small text below the title, with nice clarifications. It doesn't really support it, it can be done with HTML tags in the labels but it doesn't seem intended. I got it working in the dynamically-generated version _or_ the SVG exported version, but not both.

I suppose all we want is the dynamically generated version, which we can style, so it's not a huge deal. I'll think about it.

Reply:
we use mermaid extensively and it is compatible with github, I'd sacrifice a bit of expressiveness to have update-ability of the graphs. amazing things can be created https://github.com/farcasterxyz/hub/blob/main/README.md